src/_guides/language/language-tour.md

```diff
@@ -634,6 +634,16 @@ Dart list:
 var list = [1, 2, 3];
 {% endprettify %}
 
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  If you are wondering whether this list literal is
+  [strong mode compliant](/guides/language/sound-dart), it is. Thanks to
+  [type inference](/guides/language/sound-dart#type-inference),
+  the analyzer infers a list of type `int`.
+  If you tried to add another type to this list under strong mode,
```

Comment:
Upon further reflection, "add another type" sounds wrong to me. Maybe "non-int objects"? Because you're adding objects, not types.